Protos/V1/sessions_common.proto

```diff
@@ -9,130 +12,142 @@ import "session_status.proto";
+  google.protobuf.Timestamp cancelled_at = 4; /** The cancellation date. Only set when status is 'cancelled'. */
 }
 
 /**
- * Request for listing sessions
+ * Request to list sessions.
+ *
+ * Use pagination, filtering and sorting.
  */
 message ListSessionsRequest {
-  // Page number
-  int32 page = 1;
-  // Page size
-  int32 page_size = 2;
-
-  // To filter returned sessions
+  int32 page = 1; /** The page number. Start at 0. */
+  int32 page_size = 2; /** The page size. */
+
+  /**
+   * Represents a filter with every available fields.
+   *
+   * Every field can be used as the same time.
+   */
   message Filter {
-    // Name of the application
-    string application_name = 1;
-    // Version of the application
-    string application_version = 2;
-    // Id of the session
-    string session_id = 3;
-    // Date after the creation of the session
-    google.protobuf.Timestamp created_after = 4;
-    // Date before the creation of the session
-    google.protobuf.Timestamp created_before = 5;
-    // Date after the cancellation of the session
-    google.protobuf.Timestamp cancelled_after = 6;
-    // Date before the cancellation of the session
-    google.protobuf.Timestamp cancelled_before = 7;
-    // Status of the session
-    session_status.SessionStatus status = 8;
+    string application_name = 1; /** The application name. */
+    string application_version = 2; /** The application version. */
+    string session_id = 3; /** The session ID. */
+    google.protobuf.Timestamp created_after = 4; /** Use the creation date of a session to filter sessions created after the input. */
+    google.protobuf.Timestamp created_before = 5; /** Use the creation date of a session to filter sessions created before the input. */
+    google.protobuf.Timestamp cancelled_after = 6; /** Use the cancellation date of a session to filter sessions cancelled after the input. */
+    google.protobuf.Timestamp cancelled_before = 7; /** Use the cancellation date of a session to filter sessions cancelled before the input. */
+    session_status.SessionStatus status = 8; /** The session status. */
   }
 
-  // Filter to apply to the returned sessions
+  /** The filter.
+   *
+   * Must be set for every request but can be empty.
+   */
   Filter filter = 3;
 
-  // Available fields to order the returned sessions
+  /**
+   * Represents every available field to use in order to sort sessions.
+   */
   enum OrderByField {
-    ORDER_BY_FIELD_UNSPECIFIED = 0;
-    ORDER_BY_FIELD_SESSION_ID = 1;
-    ORDER_BY_FIELD_STATUS = 2;
-    ORDER_BY_FIELD_CREATED_AT = 3;
-    ORDER_BY_FIELD_CANCELLED_AT = 4;
+    ORDER_BY_FIELD_UNSPECIFIED = 0; /** Unspecified. */
+    ORDER_BY_FIELD_SESSION_ID = 1; /** The session ID. */
+    ORDER_BY_FIELD_STATUS = 2; /** The session status. */
+    ORDER_BY_FIELD_CREATED_AT = 3; /** The creation date. */
+    ORDER_BY_FIELD_CANCELLED_AT = 4; /** The cancellation date. */
```

Comment:
We are still using cancellation and cancelation, see e.g., task_status O_O

Reply:
That's my fault. Will do a PR to fix it.